include/zephyr/bluetooth/iso.h

```diff
@@ -739,6 +751,22 @@ int bt_iso_chan_get_info(const struct bt_iso_chan *chan,
  */
 enum bt_iso_chan_type bt_iso_chan_get_type(const struct bt_iso_chan *chan);
 
+/** @brief Get ISO transmission timing info
+ *
+ *  @details Reads timing information for transmitted ISO packet on CIS or BIS.
```

Comment:
Suggest to use the term used by this file already

```suggestion
 *  @details Reads timing information for transmitted ISO packet on an ISO channel.
```